live_validation_scripts/validate_live_pixel.mjs

```diff
@@ -3,61 +3,578 @@
+    [PixelValidationResult.VALIDATION_PASSED]: new Set(),
+};
+const unusedPixelDefintions = new Set();
+const pixelValidationResults = new Map();
+
+const stats = {
+    numPixelDefinitionFiles: 0,
+    numPixelDefinitions: 0,
+
+    numValidOwners: 0,
+    numPixelOwnersWithErrors: 0,
+
+    totalRows: 0,
+
+    // totalRows = documentedRows + undocumentedRows
+    documentedRows: 0,
+    undocumentedRows: 0,
+
+    totalPixels: 0,
+
+    // totalPixels = documentedPixels + undocumentedPixels
+    undocumentedPixels: 0,
+    documentedPixels: 0,
+
+    // totalPixels = accessedPixels + unaccesssedPixels
+    accessedPixels: 0,
+    // unaccesssedPixels only occur when a documented pixel is not accessed in the validation process
+    // we only learn about undocumented pixels if they are accessed
+    unaccesssedPixels: 0,
+
+    uniquePixelParamVariants: 0,
+
+    // While each pixel can have multiple validation results
+    // Each row has a single validation result
+    // Sum over uniquePerSet does not need to equal totalPixels
+    // Sum over referencesPerSet does need to equal totalRows
+    numSets: 0,
+    uniquePerSet: {
+        [PixelValidationResult.UNDOCUMENTED]: 0,
+        [PixelValidationResult.OLD_APP_VERSION]: 0,
+        [PixelValidationResult.VALIDATION_FAILED]: 0,
+        [PixelValidationResult.VALIDATION_PASSED]: 0,
+    },
+    referencesPerSet: {
+        [PixelValidationResult.UNDOCUMENTED]: 0,
+        [PixelValidationResult.OLD_APP_VERSION]: 0,
+        [PixelValidationResult.VALIDATION_FAILED]: 0,
+        [PixelValidationResult.VALIDATION_PASSED]: 0,
+    },
+};
+
+function getArgParser(description) {
+    return yargs(hideBin(process.argv))
+        .command('$0 csvFile dirPath userMapFile', description, (yargs) => {
+            return yargs
+                .positional('csvFile', {
+                    describe: 'Path to CSV file containing pixels to validate ',
+                    type: 'string',
+                    demandOption: true,
+                })
+                .positional('dirPath', {
+                    describe: 'path to directory containing the pixels folder and common_[params/suffixes].json in the root',
+                    type: 'string',
+                    demandOption: true,
+                    coerce: (dirPath) => {
+                        if (!fs.existsSync(dirPath) || !fs.statSync(dirPath).isDirectory()) {
+                            throw new Error(`Directory path ${dirPath} does not exist!`);
+                        }
+                        return dirPath;
+                    },
+                })
+                .positional('userMapFile', {
+                    describe: 'Path to user map YAML file',
+                    type: 'string',
+                    demandOption: true,
+                });
+        })
+        .demandOption('dirPath');
+}
+
+const argv = getArgParser('Validate live pixels').parse();
+
+// Produces userMap
+function readUserMap(userMapFile) {
```

Comment:
Minor: could probably move to `file_utils.mjs` and be used here and in `validate_schema.mjs`